mathesar/api/exceptions/database_exceptions/exceptions.py

```diff
@@ -203,3 +203,20 @@ def __init__(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR
     ):
         super().__init__(exception, self.error_code, message, field, details, status_code)
+
+
+class NotNullViolationAPIException(MathesarAPIException):
+    """
+    Exception raised when trying to add not null constraint to column with null value
+     or when trying to add non-null value to a column with not null constraint
+    """
+    error_code = ErrorCodes.NotNullViolation.value
+
+    def __init__(self, exception,
```

Comment:
Please use the formatting of one line per argument per the other code review thread.